tests/integration/backward_compatible/sql_test.py

```diff
@@ -151,9 +164,14 @@ def test_execute_statement_with_cursor_buffer_size(self):
         statement.cursor_buffer_size = 3
         result = self.client.sql.execute_statement(statement)
 
-        six.assertCountEqual(
-            self, [i for i in range(entry_count)], [row.get_object("age") for row in result]
-        )
+        with patch.object(result, "_fetch_next_page", wraps=result._fetch_next_page) as patched:
```

Comment:
not sure but looks like wraps is not a keyword argument on patch.object https://docs.python.org/3/library/unittest.mock.html#patch-object

```suggestion
        with patch.object(result, "_fetch_next_page") as patched:
```

Reply:
It is one of the kwargs